snap/snapcraft.yaml

```diff
@@ -407,7 +409,7 @@ parts:
         PROJECT_VERSION=local-dev
       fi
       GIT_REVISION=$(git rev-parse --short HEAD)
-      snapcraftctl set-version ${PROJECT_VERSION}-$(date +%Y%m%d)+${GIT_REVISION}
+      snapcraftctl set-version ${PROJECT_VERSION}
```

Comment:
perhaps also call out this and the postgresql change in the PR description too? and thanks for making those things specific, separate commits